cardano-cli/src/Cardano/CLI/Types/Errors/GovernanceCmdError.hs

```diff
@@ -114,5 +121,13 @@ instance Error GovernanceCmdError where
       "MIR certificates are not supported in Conway era onwards."
     GovernanceCmdGenesisDelegationNotSupportedInConway ->
       "Genesis delegation is not supported in Conway era onwards."
+    GovernanceCmdMismatchedDRepMetadataHashError expectedHash actualHash ->
+      "DRep metadata Hashes do not match!"
+        <> "\nExpected:"
+          <+> pretty (show (L.extractHash expectedHash))
+        <> "\n  Actual:"
```

Comment:
```suggestion
        <> "\nActual:"
```

Bit weird to indent the `Actual` case, whereas you don't indent the `Expected` case?

Reply:
Yes, a bit, but it is on purpouse to align the hashes so that they are easier to compare visually. I could keep the "Actual:" to the left and only align the hash, but I think it probably looks nicer like this

Reply:
Ah I see, makes sense :+1: Thanks @palas for the explanation :pray: